docs-src/tut-7-rpc.scrbl

```diff
@@ -213,123 +184,42 @@ invoke our @pyin{main} function.
+into the directory where you saved @tt{index.py}.
 
+Now open a terminal in that directory and type:
 @verbatim{
-$  REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
-   REACH_RPC_KEY=YOUR-PRESHARED-KEY \
-   ./reach server
-
-===============================
-using rpc key:
-REACH_RPC_KEY=YOUR-PRESHARED-KEY
-===============================
-
-Verifying knowledge assertions
-Verifying for generic connector
-  Verifying when ALL participants are honest
-  Verifying when NO participants are honest
-  Verifying when ONLY "Alice" is honest
-  Verifying when ONLY "Bob" is honest
-Checked 93 theorems; No failures!
-Creating network "rps-python_default" with the default driver
-Creating rps-python_ethereum-devnet_1 ... done
-
-> @"@reach-sh/rpc-server@" app /app
-> node --experimental-modules --unhandled-rejections=strict index.mjs
-
-(node:26) ExperimentalWarning: The ESM module loader is experimental.
+  $ ./reach server
 }
 
-The server is now awaiting connections.
-
-@(hrule)
-
-Open another terminal in your @tt{rps-python} directory and type the following
-to play a game of @|RPS|:
-
 @margin-note{
-Alternatively, if you would prefer to supply configuration as arguments to the
-@pyin{mk_rpc} function at runtime instead of using environment variables, this
-also works:
+Deploying your @|DApp| into production requires obtaining a certificate which
+is specific to your DNS domain and which has been signed by a certificate
+authority such as
+@link["https://letsencrypt.org/getting-started/"]{Let's Encrypt}.
 
-@verbatim{
-rpc, rpc_callbacks = \
-    mk_rpc(dict(
-  host='127.0.0.1',
-  port='3000',
-  verify='0',
-  timeout='7',
-  key='YOUR-PRESHARED-KEY',
-))
+Users who are ready to go live should consult the @seclink{ref-usage-server}
+reference section for configuration details.
 }
 
+This will launch an RPC server using the development @tech{API key}
+@tt{"opensesame"} and a premade TLS certificate.
 
-You may choose whichever best suits your use case.
+The server is now awaiting connections.
 
-Combining both methods is also possible, but settings supplied as arguments to
-@pyin{mk_rpc} are given priority over settings defined as environment
-variables.
+Open another terminal in the same directory and use @tt{./reach rpc-dev} to
+play a game of @|RPS|:
 
-Read @seclink{ref-backends-rpc-opts} for more details.
+@margin-note{
+Consult the @seclink{ref-usage-rpc-dev} reference section for more details on
+how this sub-command works.
 }
 
 @verbatim{
-$ . ./venv/bin/activate
-$ REACH_RPC_SERVER=127.0.0.1 \
-  REACH_RPC_PORT=3000 \
-  REACH_RPC_KEY=YOUR-PRESHARED-KEY \
-  REACH_RPC_TLS_REJECT_UNVERIFIED=0 \
-  python3 -u ./index.py
-
-*** Warning! TLS verification disabled! ***
-
- This is highly insecure in Real Life™ applications and must
- only be permitted under controlled conditions (such as
- during development).
+$ [ -d ./venv ] || python3 -m venv ./venv
+$ source ./venv/bin/activate
+$ pip install --upgrade reach-rpc-client
```

Comment:
I think you should use `@cmd` and separate these two by first saying, "Install the Reach Python RPC client" and then "run it"

Reply:
Resolved in 0289e8495254ef2b2a1f33c567422bc1fade9a57.